deps.bzl

```diff
@@ -1,19 +1,33 @@
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+def _shellcheck_dependencies_impl(repository_ctx):
+    version = "v0.7.1"
```

Comment:
I was thinking that it may be really nice to allow users to specify the version, but that would mean that we need to store shas for all supported versions, we can revisit this later.